docs/proposals/perf.md

```diff
@@ -0,0 +1,364 @@
+and waits until all the pods come up and **Running**.
+
+#### K8S Master
+| Subject                        | Description                                  |
+|--------------------------------|----------------------------------------------|
+| OS                             |  Ubuntu 18.04 server 64bit                   |
+| K8S Version                    |  v1.13.5                                     |
+| Docker Version                 |  v17.09                                      |
+| CPU                            |  32vCPUs                                     |
+| RAM                            |  128GB                                       |
+| Disk Size                      |  40GB                                        |
+| Count                          |  1                                           |
+
+This VM is used to run K8S Master Services including K8S API Server and K8S Scheduler and so on.
+
+#### KubeEdge Cloud Part
+| Subject                        | Description                                  |
+|--------------------------------|----------------------------------------------|
+| OS                             |  Ubuntu 18.04 server 64bit                   |
+| Docker Version                 |  v17.09                                      |
+| CPU                            |  32vCPUs                                     |
+| RAM                            |  128GB                                       |
+| Disk Size                      |  40GB                                        |
+| Count                          |  1                                           |
+
+This VM is used to run KubeEdge Cloud Part Services including Edge Controller and Cloud Hub and so on.
+
+#### K8S Nodes
+| Subject                        | Description                                  |
+|--------------------------------|----------------------------------------------|
+| OS                             |  Ubuntu 18.04 server 64bit                   |
+| Docker Version                 |  v17.09                                      |
+| CPU                            |  32vCPUs                                     |
+| RAM                            |  128GB                                       |
+| Disk Size                      |  40GB                                        |
+| Count                          |  1...N                                       |
+
+These VMs are used to deploy numbers of KubeEdge Edge Nodes pods which are running Edged and EdgeHub and so on.
+We will adjust the Count of VMs based on the KubeEdge Edge Nodes numbers.
+
+KubeEdge Performance Test setup is similar with K8S KubeMark setup,
+where they simulate numbers of hollow-nodes in pods and deploy on K8S Cluster.
+In KubeEdge we also do the similar kind of simulation for creating KubeEdge Edge Nodes
+in pods and deploy the pods through deployment. Our pod takes up resources as below:
```

Comment:
You may call the kubeEdge nodes running as pods an "edge-node-pod", and a pod running applications an "application-pod" to avoid confusion.

Reply:
Thanks. The sentence is updated